DataProc/load.py

```diff
@@ -0,0 +1,25 @@
+import mypy
```

Comment:
I would suggest grouping third party libraries together. Same with standard modules, and our modules. Overall - 3 blocks of imports at most.